source/command_bus.coffee

```diff
@@ -7,19 +7,22 @@ class Space.messaging.CommandBus extends Space.Object
   }
 
   _handlers: null
+  _onSendHooks: null
 
   constructor: ->
     super
     @_handlers = {}
+    @_onSendHooks = []
 
-  send: (command, callback) ->
+  send: (command, callback, options={}) ->
     if @meteor.isServer
       # ON THE SERVER
       handler = @_handlers[command.typeName()]
-      if not handler?
+      hook(command) for hook in @_onSendHooks when !options.ignoreHooks
```

Comment:
Take a look here, where it is used: https://github.com/meteor-space/testing/blob/develop/source/bdd/aggregates-bdd-api.coffee#L76